autorelease/kokoro.py

```diff
@@ -42,13 +43,18 @@ def _send_pubsub_message(
     return resp
 
 
-def _make_build_request(job_name: str, sha: str, env_vars: dict = None) -> str:
+def _make_build_request(
+    job_name: str, sha: str, env_vars: dict = None, multi_scm_name: Optional[str] = None
+) -> str:
     request = kokoro_api_pb2.BuildRequest(
         full_job_name=job_name,
-        scm_revision=kokoro_api_pb2.ScmRevision(
-            github_scm_revision=kokoro_api_pb2.GithubScmRevision(commit_sha=sha)
-        ),
     )
+    if multi_scm_name:
```

Comment:
Please add a comment explaining why the code does what it does.

Reply:
Done.